tests/gold_tests/tls_hooks/tls_hooks18.test.py

```diff
@@ -56,23 +54,23 @@
 
 Test.PrepareTestPlugin(os.path.join(Test.Variables.AtsTestPluginsDir, 'ssl_hook_test.so'), ts, '-client_hello=2')
 
-tr = Test.AddTestRun("Test two client hello hooks")
+tr=Test.AddTestRun("Test two client hello hooks")
 tr.Processes.Default.StartBefore(server)
 tr.Processes.Default.StartBefore(Test.Processes.ts)
-tr.StillRunningAfter = ts
-tr.StillRunningAfter = server
-tr.Processes.Default.Command = 'curl -k -H \'host:example.com:{0}\' https://127.0.0.1:{0}'.format(ts.Variables.ssl_port)
-tr.Processes.Default.ReturnCode = 0
-tr.Processes.Default.Streams.stdout = "gold/preaccept-1.gold"
+tr.StillRunningAfter=ts
+tr.StillRunningAfter=server
+tr.Processes.Default.Command='curl -k -H \'host:example.com:{0}\' https://127.0.0.1:{0}'.format(ts.Variables.ssl_port)
+tr.Processes.Default.ReturnCode=0
+tr.Processes.Default.Streams.stdout="gold/preaccept-1.gold"
```

Comment:
Interesting. Is this spacing is accepted by pep 8? Normally for variable instantiations like this it prefers spaces around the `=`.

Reply:
agreed! This was the committed output from autopep8 target.

Reply:
Oh, I think I see what's happening. If you check out line 45, I think that has a mistake in the patch that is throwing off the parser:

https://github.com/apache/trafficserver/blob/7504f6726fed96e1b11b5bad1348af4932d1c202/tests/gold_tests/tls_hooks/tls_hooks18.test.py#L45

I'll create a patch.

Reply:
Here you go: #7643

Reply:
good find! How did this test pass?